index.html

```diff
@@ -1,60 +1,87 @@
-	<h4>Create an intuit account</h4>
-	<p>One account for everything intuit, including Mint</p>
-	<a href="#">Learn more</a>
-	<p>Already have an account?</p><span><a href="#"> Sign in</a></span>
-   <label for="email">Email address
-	<input type="email" name="email" id="email">
-   </label>
-   <label for="phone-number">Phone(recommended)
-   	<input type="number" name="phone-number" id="phone-number">
-   </label>
-   <p>Standard call, messaging or data rates may apply</p>
-   <label for="user-password">Password
-   	<input type="password" name="password" id="user-password">
-   </label>
-   <button class="btn"><i class="fa fa-lock"> Create Account</i></button>
-   <p>By clicking Create Account, you agree to our <a href="#">Terms</a> and have read and acknowledge our<a href="#">Privacy Statement.</a></p>
+			<fieldset class="collect-data">
+				<div class="information">
+					<h4>Create an intuit account</h4>
+					<p>One account for everything intuit, including Mint</p>
+					<a href="#">Learn more</a>
+				</div>
+				<form class="my-data" action="">
+					<div>
+						<p><label for="email">Email address </label></p>
+						<p><input type="email" name="email" id="email" /></p>
+					</div>
+					<div>
+						<p><label for="phone-number">Phone(recommended) </label></p>
+						<p><input type="number" name="phone-number" id="phone-number"></p>
+					</div>
+					<p class="messaging">Standard call, messaging or data rates may apply</p>
+					<div>
+						<p><label for="user-password">Password</label></p>
+						<p><input type="password" name="password" id="user-password"></p>
+					</div>
 
-   <!-- footer inside the form -->
+					<button class="btn-create"><i class="fa fa-lock"> Create Account</i></button>
+				</form>
 
-   <footer>
-   	<small> invisible reCAPTCHA by Google <a href="#">Privacy Policy</a> and <a href="#">Terms of Use</a></small>
-   </footer>
+				<div class="final-form-data">
+					<p><i>By clicking Create Account, you agree to our <a href="#">Terms</a> and have read and acknowledge our<a href="#">Privacy Statement.</a></i></p>
+				</div>
 
-</fieldset>
-<!-- Legal Privacy and Security section-->
-<div class="bottom-section">
-	<p>Legal  Privacy  Security</p>
-	<p>&#169;2019 Intuit, Inc.All rights reserved. Intuit, QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.</p>
-	<p>Terms and conditions, features, support, pricing, and service options subject to change without notice.</p>
-</div>
+				<!-- form footer -->
+				<footer class="footer-google">
+					<small class="invisible"> invisible reCAPTCHA by Google <a href="#">Privacy Policy</a> and <a href="#">Terms of Use</a></small>
+				</footer>
+			</fieldset>
 
-</div>
-</body>
+			<!-- Legal Privacy and Security section-->
+			<div class="bottom-section">
+				<p class="footer-links">
+					<ul>
+						<li><a href="#">Legal</a></li>
+						<li><a href="#">Privacy</a></li>
+						<li><a href="#">Security</a></li>
+					</ul>
+				</p>
```

Comment:
`SUGGESTION`

Consider using a `<div>` instead of `<p>`, a `<p>` element can not be a parent elelemnt to `<ul>` elelement.